numbagg/transform.py

```diff
@@ -45,15 +45,14 @@ def _apply_ast_rewrite(func, node_transformer):
     tree = ast.parse(orig_source)
     tree = node_transformer.visit(tree)
     ast.fix_missing_locations(tree)
-    source = compile(tree, filename='<ast>', mode='exec')
+    source = compile(tree, filename="<ast>", mode="exec")
 
     scope = {}
     exec(source, _func_globals(func), scope)
     try:
         return scope[_TRANFORMED_FUNC_NAME]
     except KeyError:
-        raise TypeError('failed to rewrite function definition:\n%s'
-                        % orig_source)
+        raise TypeError("failed to rewrite function definition:\n%s" % orig_source)
 
 
 def _ast_arg(name):
```

Comment:
we can clean up this function if we don't care about Python 2 support (and we definitely do not care about Python 2 anymore!)